build/olm-catalog/template.yaml

```diff
@@ -23,8 +22,28 @@ spec:
   version: 0.9.0
   replaces: ember-csi-operator.v0.0.1
   minKubeVersion: 1.13.0
-  description: |-
-     Operator to deploy Ember-CSI, a multi-vendor CSI plugin driver supporting over 80 storage drivers in a single plugin to provide block and mount storage to container orchestration systems.
+  description: |
+    Ember-CSI is a multi-vendor CSI plugin driver supporting
+    over 80 storage drivers in a single plugin to provide block and mount
+    storage to container orchestration systems.
+
+    The operator adds support for management of multiple Ember CSI instances
+    within a cluster and simplifies storage backend deployments.
+
+    ### Getting started
+    Please have a look at our quick start documentation on how to get started:
+    [github.com/embercsi/ember-csi-operator#quick-start](https://github.com/embercsi/ember-csi-operator#quick-start).
```

Comment:
Should we mention that we recommend using the form instead using the YAML on the installation?

Reply:
Good point, will add that

Reply:
Done